From 5345ed805ac5000eaf41d0fad7aad6158dbcf814 Mon Sep 17 00:00:00 2001
From: Charles Mirabile <cmirabil@redhat.com>
Date: Tue, 22 Apr 2025 11:40:42 -0400
Subject: [PATCH 53/74] fixup! drivers: hwmon: PAC1934: Add PAC1934 driver

replace non-existant sense_resistances property reference with shunt_resistors that is actually in dts
---
 drivers/hwmon/pac193x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hwmon/pac193x.c b/drivers/hwmon/pac193x.c
index 8a9ca9ec07b5..6a7c7ad93b91 100644
--- a/drivers/hwmon/pac193x.c
+++ b/drivers/hwmon/pac193x.c
@@ -761,7 +761,7 @@ static int pac193x_probe(struct i2c_client *client)
 		dev_err(dev, "can not get energy_acc_count:%d\n", ret);
 		data->energy_acc_count = 0;
 	}
-	ret = of_property_read_u32_array(dev->of_node, "sense_resistances",
+	ret = of_property_read_u32_array(dev->of_node, "shunt_resistors",
 					 data->sense_resistances,
 					 PAC193X_MAX_CHAN_CNT);
 	if (0 != ret) {
-- 
2.52.0

